web/src/components/tasks/business-formation/BusinessFormationPaginator.tsx

```diff
@@ -48,12 +48,34 @@ export const BusinessFormationPaginator = (): ReactElement => {
 
   type ConfigFormationFields = keyof typeof Config.formation.fields;
 
+  const determineStepsWithErrors = (overrides?: {
+    hasSubmitted: boolean;
+  }): { stepsWithErrors: StepperStep[]; stepStates: StepperStep[] } => {
+    const stepStates = BusinessFormationStepsConfiguration.map((value) => {
+      return {
+        name: value.name,
+        hasError: doesStepHaveError(value.name, overrides),
+        isComplete: isStepCompleted(value.name),
+      };
+    });
+
+    const stepsWithErrors = stepStates.filter((stepState) => {
+      return stepState.hasError;
+    });
+    return { stepsWithErrors, stepStates };
+  };
+
+  const { stepsWithErrors, stepStates } = determineStepsWithErrors();
+
+  const selectedStepHasErrors =
+    (state.hasBeenSubmitted && stepStates[state.stepIndex].hasError && state.stepIndex !== 0) ?? // so this inparticular is needed because the name check errors are inline while everywhere else are banners that only appear after submittion
+    false;
```

Comment:
sand - so this in-particular is needed because the name check errors are inline while everywhere else are banners that only appear after submission

This is probably an inconsistency I should bring up with design as it's kind of a pain to maintain and unnecessarily complicates things